test/unit/variable-scope.test.js

```diff
@@ -422,4 +422,79 @@ describe('VariableScope', function () {
             });
         });
     });
+
+    describe('._addLayer()', function () {
+        var layerOne = new VariableList({}, [{
+                key: 'var-1-layerOne',
+                value: 'var-1-layerOne-value'
+            }, {
+                key: 'var-2-layerOne',
+                value: 'var-2-layerOne-value'
+            }]),
+            layerTwo = new VariableList({}, [{
+                key: 'var-1-layerTwo',
+                value: 'var-1-layerTwo-value'
+            }, {
+                key: 'var-2-layerTwo',
+                value: 'var-2-layerTwo-value'
+            }]);
+
+        it('adds a variable list to the current scope', function () {
+            var scope = new VariableScope(layerOne);
+            scope._addLayer(layerTwo);
+
+            expect(scope.layers.length).to.be(1);
+            expect(VariableList.isVariableList(scope.layers[0])).to.be.ok();
```

Comment:
I don't think that's required as the two `lists` are being store separately. The one initialized in the constructor is `this.values` and the layers are added as a private property as `this._layers`